Source/Modeling/Plugins/JoistAreaFeatures/CreationFeature.cs

```diff
@@ -103,7 +103,7 @@ private void DetachHandlers()
         private void Polygon_OnPreviewRequested(object sender, ToleratedObjectEventArgs eventArgs)
         {
             Graphics.Clear();
-            var currentAppliedValues = GetAppliedAttributes(Component);
+            //var currentAppliedValues = GetAppliedAttributes(Component);     //this method seems to have been removed
 
```

Comment:
Then add call to get values from Component via extension method direct so data is refreshed please.

Reply:
the variable is not used elsewhere